src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py

```diff
@@ -381,8 +381,6 @@ def _display_contour(self, obj, plotter: Union[BackgroundPlotter, pv.Plotter]):
                         plotter.add_mesh(mesh.contour(isosurfaces=20))
 
     def _fetch_surface(self, obj):
-        surface_api = obj._api_helper.surface_api
```

Comment:
This is an update which is required for multiple surface creation. This was caught during Py-Console integration and corrected.